config/routes.rb

```diff
@@ -1,8 +1,11 @@
 Rails.application.routes.draw do
+  resources :comments
   resources :media
   resources :authorships
   mount_devise_token_auth_for 'User', at: 'auth'
-  resources :users
+  resources :users do
+    resources :comments
```

Comment:
Remove this nested route. We'll add it back when we actually need it. But in the actual CFA, we load everything literally (except articles) so I don't see myself using this.

Reply:
Moreover, for CUD operations on comments, I'm already doing that through articles, never through users.